chef-config/lib/chef-config/config.rb

```diff
@@ -690,6 +690,11 @@ def self.windows_home_path
       default :watchdog_timeout, 2 * (60 * 60) # 2 hours
     end
 
+    # Only use fips compliant algorithms
+    default( :fips_mode ) do
```

Comment:
This should be `default( :openssl_fips_mode), OpenSSL::OPENSSL_FIPS`

We aren't actually building "fips_mode" for Chef, this is just a "can we use MD5 or not?" switch for the client.  What the client would need to do for "fips_mode" would be massively more extensive, and we don't want to even think about starting to build that yet without having an audit to tell us what work would need to be done.  And calling what we're building now "fips_mode" would be false advertising to people who read it in the config.